libcob/common.h

```diff
@@ -108,6 +108,11 @@ typedef __mpz_struct mpz_t[1];
 #define	cob_screen_ptr		cob_screen *
 #define	cob_file_key_ptr	cob_file_key *
 
+/* For use as booleans */
```

Comment:
bool is something for 4.x with stdbool, but we don't need that for this PR, so please drop

Reply:
done